[2022-02-01 15:51:14,664] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: first_dag.delay_function_execute scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-01 15:51:14,675] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: first_dag.delay_function_execute scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-01 15:51:14,675] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-01 15:51:14,675] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-01 15:51:14,675] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-01 15:51:14,686] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): delay_function_execute> on 2022-01-31 00:00:00+00:00
[2022-02-01 15:51:14,693] {standard_task_runner.py:52} INFO - Started process 93 to run task
[2022-02-01 15:51:14,697] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'delay_function_execute', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '948', '--raw', '--subdir', 'DAGS_FOLDER/firstdag.py', '--cfg-path', '/tmp/tmpl1s9vs53', '--error-file', '/tmp/tmp9an1kiih']
[2022-02-01 15:51:14,697] {standard_task_runner.py:77} INFO - Job 948: Subtask delay_function_execute
[2022-02-01 15:51:14,781] {logging_mixin.py:109} INFO - Running <TaskInstance: first_dag.delay_function_execute scheduled__2022-01-31T00:00:00+00:00 [running]> on host 846ca9424c84
[2022-02-01 15:51:14,906] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=delay_function_execute
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-02-01 15:51:14,907] {logging_mixin.py:109} INFO - starting delay for 5 sec
[2022-02-01 15:51:19,912] {logging_mixin.py:109} INFO - 5 sec ends
[2022-02-01 15:51:19,912] {python.py:175} INFO - Done. Returned value was: dealy function ends
[2022-02-01 15:51:19,940] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=delay_function_execute, execution_date=20220131T000000, start_date=20220201T155114, end_date=20220201T155119
[2022-02-01 15:51:19,999] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-01 15:51:20,041] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
